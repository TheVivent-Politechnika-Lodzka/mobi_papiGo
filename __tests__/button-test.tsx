import 'react-native';
import React from 'react';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';
import Button from '../src/components/button';

it('Button render test', () => {
  const button = renderer
    .create(<Button value="test" onPress={() => {}} />)
    .toJSON();
  expect(button).toMatchSnapshot();
});
