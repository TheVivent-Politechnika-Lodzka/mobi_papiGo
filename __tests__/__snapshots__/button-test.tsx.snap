// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button render test 1`] = `
<View
  accessible={true}
  collapsable={false}
  focusable={true}
  onBlur={[Function]}
  onClick={[Function]}
  onFocus={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "backgroundColor": "#009688",
      "borderRadius": 10,
      "elevation": 8,
      "margin": 2,
      "paddingHorizontal": 12,
      "paddingVertical": 10,
    }
  }
>
  <Text
    style={
      Object {
        "alignSelf": "center",
        "color": "#fff",
        "fontSize": 18,
        "fontWeight": "bold",
        "textTransform": "uppercase",
      }
    }
  >
    test
  </Text>
</View>
`;
